src/pages/User/components/UserHeader/UserHeader.vue

```diff
@@ -88,6 +86,6 @@ const uploadImage = async (event: Event) => {
 };
 
 const userMembershipStatus = computed(() =>
-  props.isMembershipActive ? 'Usuario activo' : 'Usuario inactivo'
+  props.isMembershipActive ? 'Usuario' : 'Usuario'
```

Comment:
I don't like this change, please check

Reply:
@angelik0828

Reply:
@Eghinner now there is not a membership? Maybe remove this variable and just hardcode it in the code